src/Commands/FormatAnalyzersCommand.cs

```diff
@@ -34,7 +35,16 @@ public async Task<int> InvokeAsync(InvocationContext context)
                 var formatOptions = parseResult.ParseVerbosityOption(FormatOptions.Instance);
                 var logger = context.Console.SetupLogging(minimalLogLevel: formatOptions.LogLevel, minimalErrorLevel: LogLevel.Warning);
                 formatOptions = parseResult.ParseCommonOptions(formatOptions, logger);
-                formatOptions = parseResult.ParseWorkspaceOptions(formatOptions);
+
+                try
+                {
+                    formatOptions = parseResult.ParseWorkspaceOptions(formatOptions);
+                }
+                catch (FileNotFoundException fex)
+                {
+                    logger.LogError(fex.Message);
```

Comment:
shouldn't we show a message saying we couldn't find the file instead of logging an error? Doesn't this still show a stack trace to the user?

Reply:
since this is in `main` I am ok with waiting for a full re-work. do we want this to go into the 6.x branch instead?

Reply:
alright, I'll email .NET Tactics to see if they want to take this.